kea-vscode/src/views/common/file-tree-node.ts

```diff
@@ -40,6 +40,13 @@ export class FileTreeNode implements IParentTreeNode<ReviewCommentTreeNode> {
     treeItem.iconPath = this.#iconPath;
     treeItem.tooltip = this.#tooltip;
 
+    // Add command to open file diff when clicked
+    treeItem.command = {
+      command: "kea.openCommitFileDiff",
+      title: "Open File Diff",
+      arguments: [{ resourceUri: this.#resourceUri }],
```

Comment:
`kea.openCommitFileDiff` opens a file, but there isn't any difference between the left and right hand side of the diff, for any file.

Reply:
The diff for the files under the `FilesRootTreeNode` is broken - it no longer shows diffs for files. Instead, both sides are temp files in the local file system.

Reply:
This should use the new CommandManager interface you created. Please use it everywhere.